wasmbinding/stargate_whitelist.go

```diff
@@ -184,34 +185,48 @@ func init() {
 	setWhitelistedQuery("/osmosis.concentratedliquidity.v1beta1.Query/CFMMPoolIdLinkFromConcentratedPoolId", &concentratedliquidityquery.CFMMPoolIdLinkFromConcentratedPoolIdResponse{})
 }
 
-// GetWhitelistedQuery returns the whitelisted query at the provided path.
+// IsWhitelistedQuery returns if the query is not whitelisted.
+func IsWhitelistedQuery(queryPath string) error {
```

Comment:
Exposed this method in place of getWhitelistedQuery to avoid unexported usage of a function that can leak memory if not used properly